handbook/upgrade/upgrade.md

```diff
@@ -0,0 +1,280 @@
+
+1. Configure SystemLink to use the newly created MongoDB server or replica set.
+
+1. Install `nislmigrate` on your new SystemLink Server.
+
+1. Attach the `D:\` volume used to capture data from your original SystemLink server.
+
+1. Run the command `nislmigrate restore --all --secret <your secret> --dir D:\migration`.
+
+1. Verify your new SystemLink Server has all the expected migrated data.
+
+### Single Node to Multi Node with File Storage
+
+This workflow describes steps to upgrade a single node deployment of SystemLink Server to a multi node deployment where the file storage instance used by SystemLink is hosted on a a dedicated NAS, SAN, or AWS S3.
+
+1. Backup your SystemLink Server.
+
+1. Attach a volume to store data captured by `nislmigrate`. This will be referenced as `D:\`.
+
+1. Install `nislmigrate`.
+
+1. Run the command `nislmigrate capture --all --secret <your secret> --dir D:\migration`.
+
+1. Copy the files from the original file storage location to the new file store.
+
+    !!!note
+        This file copy/move is not facilitated by `nislmigrate` directly. Use File Explorer or PowerShell if migrating to a NAS or SAN. Use [AWS CLI](https://aws.amazon.com/cli/) if migrating to S3.
+
+1. Detach `D:\`.
+
+    !!!note
+        At this point in the workflow your original SystemLink Server is in an operable state. Be aware if any new data is created or ingested by SystemLink Server at this time it will not be available to your new server.
+
+1. Provision a new file store for SystemLink.
+
+1. Install and configure the same or a newer version of SystemLink Server.
+
+1. Configure SystemLink to use the newly created file storage location.
+
+1. Install `nislmigrate` on your new SystemLink Server.
+
+1. Attach the `D:\` volume used to capture data from your original SystemLink server.
+
+1. Run the command `nislmigrate restore --all --secret <your secret> --change-file-store-root <new root> --dir D:\migration`.
+
+    !!!note
+        The `--change-file-store-root` argument flag is needed to update the file meta data the new root location of your file storage. This could be a new drive letter, UNC path, or S3 URL depending on your configuration. If this step is not complete you will be able to view file meta data (name, properties, etc), but you will not be able to preview or download files.
+
+1. Verify your new SystemLink Server has all the expected migrated data.
+
+### Single Node to Multi Node with PostGreSQL
+
+This workflow describes steps to upgrade a single node deployment of SystemLink Server to a multi node deployment where the PostGreSQL instance used by the SystemLink Test Monitor service is hosted on a its own server or replica set.
+
+As of SystemLink 21.5.TODO VERSION NUMBER SystemLink supports using PostGreSQL as the database backing the Test Monitor service. This configuration is optional, but provides significant performance improvements. `nislmigrate` does not yet support migrating PostGreSQL directly.
```

Comment:
Is this TODO still necessary?

Reply:
I thought we just used PostgresSQL in 21.5, with no option for MongoDB for test data?

Reply:
Sorry, I hadn't pushed an upgrade after I got this version number from Nick. The todo should be gone now.